packages/trpc/server/routers/viewer/organizations/create.handler.ts

```diff
@@ -66,12 +66,6 @@ const vercelCreateDomain = async (domain: string) => {
 export const createHandler = async ({ input, ctx }: CreateOptions) => {
   const { slug, name, adminEmail, adminUsername, check } = input;
 
-  const userCollisions = await prisma.user.findUnique({
-    where: {
-      email: adminEmail,
-    },
-  });
-
```

Comment:
We don't need to check collisions anymore, when creating an org, if you had another profile (persona or org) this is now allowed.